lib/linguist/languages.yml

```diff
@@ -6411,6 +6409,21 @@ Smithy:
   extensions:
   - ".smithy"
   language_id: 1027892786
+Snakemake:
+  type: programming
+  group: Python
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#33c68a"
+  extensions:
+  - ".smk"
+  - ".snakefile"
+  filenames:
+  - Snakefile
+  aliases:
+  - snakefile
```

Comment:
We need a `language_id`. See [point 4](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language)